.gitlab-ci.yml

```diff
@@ -290,8 +291,11 @@ variables:
 .if_not_e2e: &if_not_e2e
   if: $RUN_KITCHEN_TESTS == "false"
 
-.if_run_e2e: &if_run_e2e
-  if: $RUN_E2E_TESTS == "true"
+.if_auto_e2e: &if_run_e2e
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Matching up the names? Also if it's used anywhere else in the config...
```suggestion
.if_auto_e2e: &if_auto_e2e
```

Reply:
Sorry, I see a use of the old name [here](https://github.com/DataDog/datadog-agent/blob/715648dbee58084bfda8d68f3506b1269492654c/.gitlab-ci.yml#L902) as well.

Reply:
Oh yes, forgot this one. I think it should be good now